### DISCUSSION TITLE
chat history

### DISCUSSION THREAD

**Discussion by cbdgftqs** [LiveHelperChat/livehelperchat#1929] [Q&A]
Hello:
I have several departments ,is it possible to set the operator in one department wont see the chat history made by front user with other operators in other departments? 
I am using the version 4.20 and department ID is passed throught js argument.
Thx!

**Comment by remdex** [comment]
Please read the following articles. It's possible to do all that with permissions...
https://doc.livehelperchat.com/docs/chat/multiple-clients
https://doc.livehelperchat.com/docs/chat/list
https://doc.livehelperchat.com/docs/users/advanced-permissions#roles
